<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_dimentions">
     <attribute name="title">
      <string>Dimentions</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="l_width">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="box_width"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_depth">
         <property name="text">
          <string>Depth</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="box_depth"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_height">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="box_height"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_jointery">
     <attribute name="title">
      <string>Jointery</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>202</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="l_sidesback_jointery">
         <property name="text">
          <string>Sides &amp; Back</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="combo_sides_back_jointery"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_sides_faceframe_jointery">
         <property name="text">
          <string>Sides &amp; Face Frame</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="combo_sides_face_frame_jointery"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_material">
     <attribute name="title">
      <string>Material</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>159</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="l_sides_thickness">
         <property name="text">
          <string>Sides</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="mat_sides_thickness"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_back_thickness">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="mat_back_thickness"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_bottom_thickness">
         <property name="text">
          <string>Bottom</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="mat_bottom_thickness"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="l_faceframe_thickness">
         <property name="text">
          <string>Face Frame</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="mat_faceframe_thickness"/>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="subtract_faceframe_from_sides">
         <property name="text">
          <string>Deduct Face Frame Thickness from Sides</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_features">
     <attribute name="title">
      <string>Features</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>191</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_7">
       <item row="0" column="0">
        <widget class="QCheckBox" name="gen_top_xpiece">
         <property name="text">
          <string>Generate Top Cross Pieces</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="l_top_xpiece_width">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="top_xpiece_width"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>128</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_8">
       <item row="0" column="0">
        <widget class="QCheckBox" name="integrated_toekick">
         <property name="text">
          <string>Integrated Toe Kick</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="l_toekick_height">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="toekick_height"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="l_toekick_depth">
           <property name="text">
            <string>Depth</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="toekick_depth"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="l_bottom_offset">
           <property name="text">
            <string>Bottom Offset</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="bottom_offset"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>291</width>
        <height>130</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_9">
       <item row="0" column="0">
        <widget class="QCheckBox" name="gen_shelf_pins">
         <property name="text">
          <string>Generate Shelf Pins</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="l_shelf_hole_spacing">
           <property name="text">
            <string>Spacing</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="shelf_hole_spacing">
           <property name="text">
            <string>0.75</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="l_shelf_hole_diameter">
           <property name="text">
            <string>Diameter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="shelf_hole_diameter">
           <property name="text">
            <string>0.25</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="l_shelf_hole_height_range">
           <property name="text">
            <string>H Rng</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="shelf_hole_height_range_start">
             <property name="text">
              <string>10.0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="shelf_hole_height_range_end">
             <property name="text">
              <string>15.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Caclulate</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="render_window">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>571</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="preview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionCabinent"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="actionMaterial"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuInsert"/>
   <addaction name="menuConfigure"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCabinent">
   <property name="text">
    <string>Cabinent</string>
   </property>
  </action>
  <action name="actionMaterial">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
